core/src/main/java/com/mapzen/android/graphics/GraphicsModule.java

```diff
@@ -34,4 +38,29 @@
   @Provides @Singleton public SceneUpdateManager providesSceneUpdateManager() {
     return new SceneUpdateManager();
   }
+
+  /**
+   * Returns the object used create {@link com.mapzen.android.graphics.model.BitmapMarker}s.
+   * @return
+   */
+  @Provides @Singleton public BitmapMarkerFactory providesBitmapMarkerFactory() {
+    return new BitmapMarkerFactory();
+  }
+
+  /**
+   * Returns the object used to generate a style string for a Tangram
+   * {@link com.mapzen.tangram.Marker}.
+   */
+  @Provides @Singleton public StyleStringGenerator providesStyleStringGenerator() {
+    return new StyleStringGenerator();
+  }
+
+  /**
+   * Returns the object used to manager markers.
+   * {@link com.mapzen.tangram.Marker}.
+   */
+  @Provides @Singleton public MarkerManager providesMarkerManager(
```

Comment:
Does android sdk support multiple map instances (tangram android does!), if so does it make sense to not make `MarkerManager` singleton? Or will every map instance gets its own `MarkerManager`.

Reply:
The SDK doesn't support map instances, this is something I've had floating around in my head as a TODO. I created a ticket for it and will add support in a future PR https://github.com/mapzen/android/issues/436